src/G2/Equiv/Induction.hs

```diff
@@ -1,65 +1,27 @@
 {-# LANGUAGE OverloadedStrings #-}
 
 module G2.Equiv.Induction
```

Comment:
Maybe we rename this to G2.Equiv.Generalize or something?  The current name no longer really makes sense.